src/ispd/application/ConsoleColors.java

```diff
@@ -0,0 +1,89 @@
+package ispd.application;
+
+public enum ConsoleColors {
+    // Reset
+    RESET("\033[0m"), // Text Reset
+
+
+    BLACK("\033[0;30m"),  // BLACK
```

Comment:
The comments after the declarations feel unnecessary, since you chose very well-detailed enum value names.

A suggestion I do have is to opt for a builder pattern in making the colors, instead of an enum, since there is a great deal of duplication in the code to provide detailed customization. Each color has 7 enum constants associated with it, for instance with red:
* RED("\033[0;31m")
* RED_BOLD("\033[1;31m")
* RED_UNDERLINED("\033[4;31m")
* RED_BACKGROUND("\033[41m")
* RED_BRIGHT("\033[0;91m")
* RED_BOLD_BRIGHT("\033[1;91m")
* RED_BACKGROUND_BRIGHT("\033[0;101m")

A builder pattern could adapt color-changing calls to something of the sorts:
`
final var color = new ConsoleColorBuilder()
.withFg().red().underlined()
.withBg().white().bright()
.build();
`
The intermediate calls would change the underlying string dynamically, and only when .build() is called would it be computed and instantiated into a ConsoleColor object. A common value like RESET could be a static field of that class, for convenience.

This change would require a lot of work so it may be better to implement this in the future.